dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -375,6 +375,9 @@ protected void processExtraRefresh(String preferredPrefix, InmemoryConfiguration
     }
 
     private boolean verifyMethodConfig(MethodConfig methodConfig, Class<?> interfaceClass, boolean ignoreInvalidMethodConfig) {
+        if (!isNeedCheckMethod()) {
+            return true;
+        }
```

Comment:
It would be better to check anyway, but warn out only instead of throw a exception.